Content.Server/Traits/TraitSystem.cs

```diff
@@ -20,45 +21,56 @@ public override void Initialize()
         SubscribeLocalEvent<PlayerSpawnCompleteEvent>(OnPlayerSpawnComplete);
     }
 
-    // When the player is spawned in, add all trait components selected during character creation
-    private void OnPlayerSpawnComplete(PlayerSpawnCompleteEvent args)
+    /// <summary>
+    /// Attempts to add traits to the entity.
+    /// </summary>
+    /// <returns></returns>
+    public bool TryAddTraits(EntityUid mob, HumanoidCharacterProfile characterProfile)
```

Comment:
Is there a less invasive way of doing this? This seems like it would make merges a nightmare.

Reply:
Not that I can think of

Reply:
When I wrote that I did not realize that `OnPlayerSpawnComplete` was private. This is not as big of a deal as I thought it was, but wait for Lank to make the final call.